apps/core/Results.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, FlatList, Text } from 'react-native';
+import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
+
+import EmptyResults from './EmptyResults';
+
+type Props = Object;
+
+export default class Results extends React.Component<Props> {
+  resultItem = ({ item }: Object) => {
+    const { flyFrom, flyTo, localDeparture, localArrival, price } = item.node;
+    return (
+      <Text
+        style={styles.listItem}
+      >{`${flyFrom} / ${localDeparture} -> ${flyTo} / ${localArrival}  | Price: ${price}`}</Text>
+    );
+  };
+
+  keyExtractor = (item: Object, index: number) => item.node.id;
+
+  renderInner = (props: Object) => {
+    const { searchItineraries } = props;
+    const resultsLength = searchItineraries.edges.length;
```

Comment:
edges could be undefined, then this line would crash. Please use type generated by relay like [this](https://github.com/kiwicom/mobile/blob/a325684cc5e2c15dc24e00ecfb60d1d301003c7f/app/MMB/src/scenes/list/cityImage/FromToRow.js#L12)

Reply:
Types and `EmptyResults` condition updated.